crates/redeem/src/lib.rs

```diff
@@ -657,6 +643,20 @@ impl<T: Config> Pallet<T> {
         status
     }
 
+    /// get current inclusion fee based on the expected number of bytes in the transaction, and
+    /// the inclusion fee rate reported by the oracle
+    pub fn get_current_inclusion_fee() -> Result<Wrapped<T>, DispatchError> {
```

Comment:
Minor nit: if you didn't move this function, there would be less code shift.

Reply:
The function is called from the integration tests and had to be marked public, that's why I moved it

Reply:
Yeah my point was that you probably didn't need to move it, just add the `pub` prefix, but it's fine.